apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -48,6 +89,36 @@ class CoursePaneButtonRow extends PureComponent<CoursePaneButtonRowProps> {
                         <Refresh />
                     </IconButton>
                 </Tooltip>
+
+                <Tooltip title="Hide Columns">
+                    <IconButton onClick={this.handleClick} className={classes.button}>
+                        <MoreVert />
+                    </IconButton>
+                </Tooltip>
+
+                {this.state.columnSelector && (
+                    <FormControl className={classes.formControl}>
+                        <Select
+                            style={{ width: '0', zIndex: -1, color: isDarkMode() ? '#303030' : '#FAFAFA' }}
+                            inputProps={{ IconComponent: () => null }} // some styling nonsense to Thanos snap away the little arrow icon: https://aguidehub.com/blog/2023-02-25-how-to-hide-arrow-from-mui-select-in-react-js/
+                            open={this.state.columnSelector}
```

Comment:
Wouldn't this always be true because, otherwise, it wouldn't be rendered?

Reply:
you damn right